bin/bb8/docker_rsync.py

```diff
@@ -57,28 +61,51 @@ def _run_rsync(self, volumes, from_path, to_path, relative):
     def _get_volume_args(self, local_volume, volume_mode):
         mounted_volume = join("/", local_volume)
         return {
-            "bb8_ssh": {"bind": "/root/.ssh", "mode": "ro"},
+            "bb8_ssh": self._ssh_volume_bind,
             local_volume: {"bind": mounted_volume, "mode": volume_mode}
         }
 
-    def _get_remote_dir(self, starport):
-        return "{user}@{addr}:{backup_location}".format(**starport)
+    def _get_host(self, starport):
+        return "{user}@{addr}".format(**starport)
+
+    def _get_target_path(self, backup_location, name):
+        template = "{backup_location}/{name}"
+        dir = template.format(name=name, backup_location=backup_location)
+        return re.sub("/+", "/", dir)
+
+    def _get_remote_dir(self, host, target_path):
```

Comment:
could we call this method `_get_remote_data_dir` just to be really clear about which directory is being returned?